bundler/src/main.rs

```diff
@@ -84,7 +85,7 @@ type CurrentBundle = Arc<RwLock<BundleFile<NoMetadata>>>;
 #[command(author, version, about, long_about= None)]
 enum Cli {
     /// Run the service providing bundle data
-    Serve(ServeArgs),
+    Serve(Box<ServeArgs>),
```

Comment:
Assuming the box was added to avoid the large variant differences, think it'd be better to just ignore that lint